src/content/developers/docs/blocks/index.md

```diff
@@ -46,6 +46,8 @@ Proof of work means the following:
 
 - Timestamp – the time when the block was mined.
 - Block number – the length of the blockchain in blocks.
+- Base Fee Per Gas - the fee per gas required for a transaction to be included in the block.
+- Burnt Fees - the amount of ETH from base fees burnt by the block.
 - Difficulty – the effort required to mine the block.
 - mixHash – a unique identifier for that block.
 - A parent hash – the unique identifier for the block that came before (this is how blocks are linked in a chain).
```

Comment:
Just noticed we were/are mixing camelCase and sentence case here. Do we have a preference?

Reply:
Not a strong preference. The Yellow Paper is in camelCase, perhaps we just mimic that?

See page 5 (Section 4.3 The Block): https://ethereum.github.io/yellowpaper/paper.pdf

Reply:
If we do go the camelCase route, might be best to wrap in code elements (like we do on the transactions page: https://ethereum.org/en/developers/docs/transactions/#whats-a-transaction

Reply:
Done